module2/src/com/epam/onedimensionalarrays/ArrayWorker.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.onedimensionalarrays;
+
+public class ArrayWorker {
+    int sumOfArrayElements = 0;
+    public void SumOfArrayElements(int[] array, int divider) {
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] % divider == 0) {
+                sumOfArrayElements += array[i];
+            }
+        }
+    }
```

Comment:
This method should be changed:
1)return int value
2)change name of method. Read the documentation - https://docs.oracle.com/javase/tutorial/java/javaOO/methods.html
```suggestion
    public int calculateSumOfArray(int[] array, int divider) {
        int sum = 0;
        for (int i = 0; i < array.length; i++) {
            if (array[i] % divider == 0) {
                sum += array[i];
            }
        }
        return sum;
    }
```